README.md

```diff
@@ -215,27 +197,27 @@ $ git remote add myrepo git@github.com:pusheen/backend-code-exercise.git
 $ git push myrepo master
 ```
 
-__4.__ Access
+**4.** Access
 
 Give Github user `cwinters` read access to your repository.
```

Comment:
Instead of having this be me should we have this be part of the communication to the candidate?

Reply:
Agreed. Worth stepping back and reviewing this process in general.